.yamato/upm-ci.yml

```diff
@@ -104,19 +124,46 @@ test_trigger:
 {% endfor %}
 {% endfor %}
 
-nightly_test_trigger:
+release_4_2_nightly_test_trigger:
   name: Nightly tests Trigger
   triggers:
     recurring:
-      - branch: master
+      - branch: release/4.2
+        frequency: daily
+        rerun: always
+  dependencies:
+    - .yamato/upm-ci.yml#pack
+      {% for editor in release_4_2_nightly_trigger_editors %}
+      {% for platform in platforms %}
+    - .yamato/upm-ci.yml#test_{{platform.name}}_{{editor.version}}
+      {% endfor %}
+      {% endfor %}
+      
+release_4_1_nightly_test_trigger:
+  name: Nightly tests Trigger
```

Comment:
include the release early on in the name so it's easy to spot on yamato, e.g. "nightly release/4.1 tests trigger"